src/mopi/runner/runner.py

```diff
@@ -43,7 +43,17 @@ def __init__(
         add_split_category_to_datasource_(self.pipeline, experiment)
         self.plugins = obligatory_plugins_begin + plugins + obligatory_plugins_end
 
-    def run(self) -> Store:
+    def run(self, pure_inference: bool = False) -> Tuple[Store, "Pipeline"]:
+        if pure_inference:
+            logger.log("💈 Loading existing models")
+            self.pipeline.load(self.plugins)
+
+            logger.log("🔮 Predicting with pipeline")
+            preds_probs = self.pipeline.predict(self.store, self.plugins)
+            self.store.set_data(Const.final_output, preds_probs)
+
+            return self.store, self.pipeline
+
```

Comment:
Either we do this (copying sections) or we add if statements and booleans everywhere. In this packaging I tried leaving as much untouched as possible, but I can change it the other way.

Reply:
if pure_inference==True returns early here and doesn't run the rest of the pipeline, maybe it's worth just moving out to a different function `infer()`?
if there's no overlap between the old `run` (we can rename it to `train`?) and `infer`, maybe its better if they're separate?

Reply:
Yes, I can do that!

Reply:
I seperated it out, check the update please @itchingpixels :)